gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/DuplicateObservationStatementMapper.java

```diff
@@ -28,29 +28,27 @@ public class DuplicateObservationStatementMapper {
     public static final int CHAR_LIMIT_FOR_TRUNCATION = 50;
     public static final String ELLIPSIS = "...";
 
-    public void mergeDuplicateObservationStatements(RCMRMT030101UKEhrExtract ehrExtract) {
+    public void mergeOrRemoveDuplicateObservationStatements(RCMRMT030101UKEhrExtract ehrExtract) {
         ehrExtract.getComponent()
                 .stream()
                 .map(RCMRMT030101UKComponent::getEhrFolder)
                 .map(RCMRMT030101UKEhrFolder::getComponent)
                 .flatMap(List::stream)
                 .map(RCMRMT030101UKComponent3::getEhrComposition)
-                .forEach(DuplicateObservationStatementMapper::mergeDuplicateObservationStatements);
+                .forEach(DuplicateObservationStatementMapper::mergeOrRemoveDuplicateObservationStatements);
     }
 
-    private static void mergeDuplicateObservationStatements(RCMRMT030101UKEhrComposition ehrComposition) {
+    private static void mergeOrRemoveDuplicateObservationStatements(RCMRMT030101UKEhrComposition ehrComposition) {
```

Comment:
Just a side note, not part of this PR - there are a lot of static methods within this class and I'm just wondering if they need to be! 🤔

Again, just making a note - not necessarily part of this PR.